internal/utils/utils.go

```diff
@@ -17,20 +17,29 @@ package utils
 import (
 	"context"
 	"fmt"
+	"regexp"
+	"strings"
 	"time"
 
 	"github.com/cilium/cilium-cli/defaults"
 )
 
+func CheckVersion(version string) bool {
+	var check = regexp.MustCompile(`(0|[1-9][0-9]*)*\.*(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-[a-zA-Z0-9]+)*\.*(0|[1-9][0-9]*)?`).MatchString
```

Comment:
Actually it works better with `?` instead  `\.*`. Let me update that.

Reply:
Yeah I was wondering about checking for whether the image legitimately exists or not too but it seemed a bit overkill. Also, it's plausible that you could define an image tag that is accessible from the cluster but not accessible from where your CLI is running (eg private mirror for your organization) so it might be just a bit too fancy.